src/theme/DocCard/index.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
```suggestion
// Why is this swizzled?
//   1. To eliminate unnecessary text truncation on card descriptions
//   2. To allow for highlighting specific cards with a different color border
// Swizzled from version 2.3.1.

import React from "react";
```

Blocking:

One thing I forgot to mention is that I would like us to specify in each swizzled component: 

1. Why it's swizzled, and
2. Which version of docusaurus it was swizzled from. 

This is helpful when we do docusaurus updates -- it helps identify which swizzled components need to be rewritten during an update, and when they do, what functionality needs to be preserved.

Feel free to rewrite the lines I suggested. You can also see [another swizzled component](https://github.com/camunda/camunda-platform-docs/pull/1794) for an example.

Reply:
Thank you for this!